### ISSUE TITLE
Folders after KeePass import visible In MyVault, new folder in MyVault and Organisation View

### ISSUE THREAD

**Issue by Novae7** [bitwarden/clients#4455]
### Steps To Reproduce

1. Import KeePass DB
2. See Folderstructure from KeePass in My vault
3. Create Subfolder while in My Vault
4. Click on Company Vault
5. See "Folder/Subfolder" in Company Vault

### Expected Result

After create a (sub)Folder i would expect that i see the Folder only per Vault

### Actual Result

Imported Folder Structure is only Visible in My Vault
New folder are visible in any vault

### Additional Context

I found the Issue #3163 and understand that newly created folders are Visible everywhere (which i dont like either)
but in my case, after Importing von KeePass i have a different Folderstructure in My Vault...

So it looks like the Application itself support the differentiation between the Vaults only the Interface dont Support a selection.

I tested it in the Webinterface and also in the Desktop Client

### Operating System

Windows

### Operating System Version

Dosnt matter

### Web Browser

Firefox

### Build Version

2023.1.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by djsmith85** [comment]
Hi @Novae7 and thank you for your report.

Could please mention which importer you used? 
Keepass2 (xml based)
or
KeepassX (csv based)

Potentially related issue: https://github.com/bitwarden/clients/issues/2795

**Comment by Novae7** [comment]
Hi @djsmith85 
I used an Keepass2 xml Based import.

**Comment by Novae7** [comment]
![grafik](https://user-images.githubusercontent.com/2822685/212064128-52678c79-918c-4666-ab36-0d8d7644d4e3.png)
![grafik](https://user-images.githubusercontent.com/2822685/212064219-d113c593-0dc3-40c9-a28b-b660c40fe78f.png)

here you can see the difference after Importing

When i now create a folder eMail/Test it shows at My Vault like this
![grafik](https://user-images.githubusercontent.com/2822685/212064534-e2bf0bfa-53d8-4ad7-9f96-79695bbf7c32.png)


And in the Org Vault like that
![grafik](https://user-images.githubusercontent.com/2822685/212064620-cd575b1a-c610-40f6-8fd9-734cfa2771b7.png)

**Comment by Diesmo** [comment]
I just created #4471 and it might be related or a duplicate of this ticket.
I found out it only affects empty folders and is not related to any import mechanic, but instead any empty folder. Also those created via the Bitwarden UI.

I also have reproducing steps in my ticket.

**Comment by Greenderella** [comment]
Hi there,

This is the expected behavior at the moment with organizational imports. 

If you want for the nested collection eMail/test to be created, you first need to create the **eMail** collection and then, on the same import, the **eMail/test** one.

Your import should look like this:

![image](https://user-images.githubusercontent.com/5418577/214840291-462d5429-b279-4597-b800-443b8dda396d.png)

Once the import process is complete you can go ahead and delete all the placeholder items:

![image](https://user-images.githubusercontent.com/5418577/214840643-e2994264-2456-457e-afb1-00cf4c2084d8.png)

If you are interested in putting up a feature request for this particular process, I recommend you sign up on our forums and search to see if this request already exists (https://community.bitwarden.com/signup). If so, you can vote for it and contribute to any discussions about it. If not, you can re-create the request there so that it can be properly tracked.

This issue will now be closed. 

Thanks!

**Comment by Novae7** [comment]
I never talked about Collections just about folders...

I also never talked about organizational imports...

Please reopen this because its about Folder not About Collections.

Thanks!

**Comment by Greenderella** [comment]
Sorry @Novae7 I misunderstood the issue, it seems like your report is the same as this one: https://github.com/bitwarden/clients/issues/4471

I will keep your issue closed to avoid any confusions, the team is already investigating this. 

Thank you for your patience. Let us know if there is anything else we can do for you.
